packages/frontend/components/Borrow/Overview.tsx

```diff
@@ -24,17 +25,7 @@ export default function Overview() {
   const [showProvider, setShowProvider] = useState(false)
 
   return (
-    <Box
-      sx={{
-        pl: { xs: "0.25rem", sm: "1rem" },
-        pr: { xs: "0.25rem", sm: "1rem" },
-        width: "100%",
-      }}
-    >
-      <p>
-        Current state: <code>...</code>
-      </p>
-
+    <Container>
```

Comment:
Are you sure you want to use Container outside of a page ? I think a box here is a better fit 🤔 

Edit: maybe we can simply wrap everything into the Grid ? Or a Paper element if you need some styling like background and border radius ? 🤔 

> The container centers your content horizontally. It's the most basic layout element.
> While containers can be nested, most layouts do not require a nested container.
> https://mui.com/material-ui/react-container/#main-content